### ISSUE TITLE
Is it typing error? (at loss function) 

### ISSUE THREAD

**Issue by sungonuni** [valeoai/ADVENT#28]
Hello. Maybe I found some difference between code and paper. 

![image](https://user-images.githubusercontent.com/68697884/122883440-2a0d8f00-d378-11eb-8968-fcf3c11d071e.png)
The loss function of training discriminator represents that source domain is 0, and target domain is 1.

![image](https://user-images.githubusercontent.com/68697884/122883290-021e2b80-d378-11eb-84b0-9c388d2c3fc8.png)
However, same part in paper shows source domain is 1, and target domain is 0.

Is it the intended notation or just typing error?
